script.js

```diff
@@ -21,4 +21,210 @@ function closeScreen() {
+                <button class="modal-12" onclick="window.location.href='${popData[data].GitHubLink}'">See Source <img src="${popData[data].githubIcon}" alt="github logo"></button>
+            </div>
+        </div>
+    </div>
+  `;
+  // Adding event listeners to respective buttons
+
+  const modalCloseButton = document.querySelector('.modal-top');
+  function toggleModal() {
+    if (getComputedStyle(modal).display === 'flex') {
+      modal.classList.add('modal-hide');
+      setTimeout(() => {
+        modal.style.display = 'none';
+        modal.classList.remove('modal-show', 'modal-hide');
+      }, 200);
+    } else {
+      modal.style.display = 'flex';
+      modal.classList.add('modal-show');
+    }
+  }
+
+  modalCloseButton.addEventListener('click', () => toggleModal());
+
+  modal.addEventListener('click', (event) => {
+    if (event.currentTarget === event.target) toggleModal();
+  });
+}
+
+// Adding work cards dynamically
+const workProfile = document.querySelector('.work-profiles-10');
+
+for (let i = 0; i < 3; i += 1) {
+  workProfile.innerHTML += `
+  <div class="work-profile-1 ${popData[i].classNO}">
+  <div class="works-text">
+      <div class="works-title">
+          <h3>
+          Multi-Post Stories<br>
+          Gain+Glory
+          </h3>
+      </div>
+      <div class="works-tags">
+          <ul class="tags">
+              <li class="tech-tag">Ruby on Rails</li>
+              <li class="tech-tag">CSS</li>
+              <li class="tech-tag">Javascript</li>
+              <li class="tech-tag">HTML</li>
+          </ul>
+      </div>
+      <div class="works-cta">
+          <button type="submit" class="project-button" onclick="modalSection(${i})">
+              See Project
+          </button>
+      </div>
+  </div>
+</div>
+  `;
+}
+
+const workProfile2 = document.querySelector('.work-profiles-20');
+
+for (let i = 3; i < popData.length; i += 1) {
+  workProfile2.innerHTML += `
+  <div class="work-profile-1 ${popData[i].classNO}">
+  <div class="works-text">
+      <div class="works-title">
+          <h3>
+          Multi-Post Stories<br>
+          Gain+Glory
+          </h3>
+      </div>
+      <div class="works-tags">
+          <ul class="tags">
+              <li class="tech-tag">Ruby on Rails</li>
+              <li class="tech-tag">CSS</li>
+              <li class="tech-tag">Javascript</li>
+              <li class="tech-tag">HTML</li>
+          </ul>
+      </div>
+      <div class="works-cta">
+          <button type="submit" class="project-button" onclick="modalSection(${i})">
+              See Project
+          </button>
+      </div>
+  </div>
+</div>
+  `;
+}
+
+const formSubmit = document.querySelector('.button-submit-1');
+
+// Checking if the email is valid
+
+const isEmailValid = (email) => {
+  if (email.match(/^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/)) {
```

Comment:
- [ ] Kindly fix the validation functionality because it's not working. Suggestion: try to test with something like `email === email.toLowerCase()` 👍 

![Animation2](https://user-images.githubusercontent.com/50721479/190707744-fc932020-9554-4e9e-9c67-2793dcbe4354.gif)